src/transformers/models/layoutlmv2/tokenization_layoutlmv2.py

```diff
@@ -893,7 +893,9 @@ def prepare_for_model(
         """
         Prepares a sequence or a pair of sequences so that it can be used by the model. It adds special tokens,
         truncates sequences if overflowing while taking into account the special tokens and manages a moving window
-        (with user defined stride) for overflowing tokens.
+        (with user defined stride) for overflowing tokens. Please Note, for `text_pair` different than `None` and
+        `truncation_strategy = longest_first` or `True`, it is not possible to return overflowing tokens. Such a
+        combination of arguments will raise an error.
```

Comment:
What's the reason the `longest_first` strategy cannot be used when providing a text pair?

Reply:
This was discussed in the PR #13179 
Three Strategies were proposed in the above PR, [Comment](https://github.com/huggingface/transformers/pull/13179#issuecomment-902669114)

Conclusion Drawn :
https://github.com/huggingface/transformers/pull/13179#issuecomment-906566964


So, I Thought same case will happen with the LayoutLmv2 as well.